src/libraries/System.Runtime.CompilerServices.Unsafe/tests/System.Runtime.CompilerServices.Unsafe.Tests.csproj

```diff
@@ -6,7 +6,7 @@
   <ItemGroup>
     <Compile Include="UnsafeTests.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net48'">
-    <ProjectReference Include="..\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp' ">
```

Comment:
Now that this is inbox in .NET Core only, it makes sense to test this for `$(NetCoreAppCurrent)` only.

Could you please delete net48 from `TargetFrameworks` and this `PackageReference`?

Reply:
Done in review feedback 3